doc/sphinx/proofs/automatic-tactics/logic.rst

```diff
@@ -166,6 +166,11 @@ Solvers for logic and equality
    You may want to use :tacn:`assert` to add some lemmas as
    hypotheses so that :tacn:`congruence` can use them.
 
+   .. tacv:: simple congruence
+      :name: simple congruence
+
+      This tactic behaves like :tacn:`congruence`, but it does not unfold definitions.
```

Comment:
```suggestion
      Behaves like :tacn:`congruence`, but does not unfold definitions.
```

Reply:
Will change.
Writing the doc I looked at [simple refine](https://coq.inria.fr/distrib/current/refman/proof-engine/tactics.html#coq:tacv.simple-refine).
Also looking at [simple destruct](https://coq.inria.fr/distrib/current/refman/proof-engine/tactics.html#coq:tacv.simple-destruct), the description very often starts with "This tactic".

Reply:
I've been removing boilerplate phrasing like "This tactic ...".  It gets tedious to read see this over and over in the hundreds of tactic and command descriptions.